content/en/network_monitoring/performance/network_page.md

```diff
@@ -38,6 +38,17 @@ You can set the timeframe over which traffic is aggregated using the time select
 
 {{< img src="network_performance_monitoring/network_page/npm_timeframe.png" alt="Time frame NPM"  style="width:30%;">}}
 
+### Cloud service autodetection
+Filtering by specific third-party services such as S3, RDS, Kinesis, and ELB can help pinpoint latency, assess database performance, and visualize your network more completely. For instance, you can filter a search query by service, view the service in the Network Map, and trace communication on that node to see affected services.
+
+{{< img src="network_performance_monitoring/network_page/cloud-service-hero-docs.png" alt="Cloud Service Map" >}}
+
+- To filter a query: in search bar(s), enter tags such as `service:s3`, `service:kinesis`, and `service:elb`. For a number of services, you can break down latency and retransmits further by using more out-of-the-box tags like `s3_bucket` and `rds_instance_type`.
+- To visualize specific managed services: in the [Network Map][1], click the dropdown next to *View* and type or select desired tags. In the map, click a particular node to view troubleshooting options.
+- To view integration metrics for a service: in the Network Page, click on a row in the [dependency table](#table). In the opened side panel, click *Integration Metrics*.
```

Comment:
the last point on integration metrics can focus more on value - e.g. use the integration metrics tab to analyze the performance of the cloud service and distinguish b/w a client-side and cloud provider-side issue